index.d.ts

```diff
@@ -0,0 +1,84 @@
+declare module 'react-native-google-cast' {
+  import * as React from 'react'
+  import { EventEmitter } from 'react-native'
+
+  export type CastDevice = {
+    id: string
+    version: string
+    name: string
+    model: string
+  }
+
+  export type CastState =
+    | 'NoDevicesAvailable'
+    | 'NotConnected'
+    | 'Connecting'
+    | 'Connected'
+
+  export type CastOptions = {
```

Comment:
nit: you may want to wrap this in a `Partial` instead of making each field optional

Reply:
Switched to `Partial` style as suggested